src/components/Details.js

```diff
@@ -0,0 +1,139 @@
+                  <p className="det_hT">
+                    {data.name}
+                  </p>
+                </div>
+                <div className="det_h">
+                  <p className="det_hN">Symbol:</p>
+                  <p className="det_hT">
+                    {data.symbol}
+                  </p>
+                </div>
+                <div className="det_h">
+                  <p className="det_hN">Ranking:</p>
+                  <p className="det_hT">
+                    #
+                    {data.market_cap_rank}
+                  </p>
+                </div>
+
+              </div>
+            </div>
+
+          </div>
+
+          <div className="det_others">
+            <div className="det_othTitle">
+              <p className="det_othN">Price</p>
+            </div>
+            <div className="det_othData">
+              <p>
+                <span className="det_span">Current:</span>
+                { ' USD ' }
+                {data2.format(data.current_price)}
+              </p>
+              <p>
+                <span className="det_span">High (last 24h):</span>
+                { ' USD ' }
+                {data2.format(data.high_24h)}
+              </p>
+              <p>
+                <span className="det_span">Low (last 24h):</span>
+                { ' USD ' }
+                {data2.format(data.low_24h)}
+              </p>
+            </div>
+          </div>
+
+          <div className="det_others d1">
+            <div className="det_othTitle">
+              <p className="det_othN">Market Capitalization</p>
+            </div>
+            <div className="det_othData">
+              <p>
+                { ' USD ' }
+                {data2.format(data.market_cap)}
+              </p>
+              <p>
+                <span className="det_span">Ranking:</span>
+                {' #'}
+                {data.market_cap_rank}
+              </p>
+            </div>
+          </div>
+
+          <div className="det_others">
+            <div className="det_othTitle">
+              <p className="det_othN">Trading Volume (last 24h)</p>
+            </div>
+            <div className="det_othData">
+              <p>
+                { ' USD ' }
+                {data2.format(data.total_volume)}
+              </p>
+            </div>
+          </div>
+
+          <div className="det_others d1">
+            <div className="det_othTitle">
+              <p className="det_othN">Maximum  Supply</p>
+            </div>
+            <div className="det_othData">
+              <p>{data2.format(data.total_supply)}</p>
+            </div>
+          </div>
+
+          <div className="det_others">
+            <div className="det_othTitle">
+              <p className="det_othN">Circulation Supply</p>
+            </div>
+            <div className="det_othData">
+              <p>{data2.format(data.circulating_supply)}</p>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] I think it would be better if you update your styling for your detail page to match the expected design. Detail texts are supposed to have **space between** as for the template design. 

### Template Design
![row_detail](https://user-images.githubusercontent.com/83042742/194680775-e4fbabf3-3e5a-453c-9680-ca66c5cee364.png)
###  Your Design
![Screenshot from 2022-10-08 03-16-59](https://user-images.githubusercontent.com/83042742/194680811-02e5aefe-7005-46db-b920-025549e3c0c1.png)

Reply:
@tresorsawasawa Done !!